browser-tests/calender-test.js

```diff
@@ -0,0 +1,38 @@
+const puppeteer = require('puppeteer');
```

Comment:
what is puppeteer used for?

Reply:
puppeteer is used to control the headless chrome process. I will sync with you offline on this